conda_smithy/feedstock_content/ci_support/disabled.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="92" height="20">
```

Comment:
So, I'm not against the idea of moving it. Unfortunately moving it around does take a bit of work. Mainly because it means updating the examples. Could I convince you to save such a change for a subsequent PR?